python_tests/test_python_frontend.py

```diff
@@ -2584,6 +2584,29 @@ def dynamic_reshape(fd: FusionDefinition) -> None:
         self.assertEqual(y.shape, torch.Size([3, 2, 2]))
         self.assertEqual(x.flatten(), y.flatten())
 
+    def test_allocation_domain_concretization(self):
+        inputs = [
+            torch.randn((0,), dtype=torch.float64, device="cuda:0").as_strided(
+                (1, 0, 1, 1), (0, 1, 1, 1)
```

Comment:
Why does the input need to be empty here?

Reply:
I thought I needed this in order to trigger concretization, i.e. `concretizeEmptyExtents` in order to enter the code path.

Reply:
I had the same question :) I think it's worth at least a code comment. It'd be even better if we can verify concretization is indeed triggered. This way, **if** later we no longer concretize empty extents, we'll get notified and change the test to trigger concretization in a different way.

Reply:
> I thought I needed this in order to trigger concretization, i.e. `concretizeEmptyExtents` in order to enter the code path.

I see. Yes that works. From C++ we could use a dynamic reshape but this way exercises the python alloc domain stuff too.